interface/forms/button_ros/new.php

```diff
@@ -0,0 +1,667 @@
+	</TR>
+	<TR class="alt">
+	<TD>
+	<B><?php echo xlt('GEN:');?></B>
+	</TD>
+	</TR>
+	<TR>
+	<TD>
+  
+		<span class="checkbox-button">
+			<input id="GEN1" name="GEN1" type="checkbox" value="on">
+			<label for="GEN1" > <?php echo xlt('Fatigue');?></label>
+		</span>
+
+		<span class="checkbox-button">
+			<input id="GEN2" name="GEN2" type="checkbox" value="on" >
+			<label for="GEN2"> <?php echo xlt('Fever');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GEN3" name="GEN3" type="checkbox" value="on" >
+			<label for="GEN3"> <?php echo xlt('Chills');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GEN4" name="GEN4" type="checkbox" value="on" >
+			<label for="GEN4"> <?php echo xlt('Night Sweats');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GEN5" name="GEN5" type="checkbox" value="on" >
+			<label for="GEN5"> <?php echo xlt('Recent Weight Gain');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GEN6" name="GEN6" type="checkbox" value="on" >
+			<label for="GEN6"> <?php echo xlt('Recent Weight Loss');?></label>
+		</span>
+		</TD>
+		</TR>
+		<TR>
+		<TD>
+		<?php echo xlt('Note abnormalities:');?> 
+					<TEXTAREA NAME="GEN_notes" ROWS=3 COLS=25 WRAP=SOFT STYLE="width: 2in; height: 0.21in"></TEXTAREA>
+					</TD>
+	</TR>	
+
+		<TR class="alt">
+	<TD>
+	<B><?php echo xlt('EYES:');?> </B>
+	</TD>
+	</TR>
+	<TR>
+	<TD>
+		<span class="checkbox-button">
+			<input id="EYES1" name="EYES1" type="checkbox" value="on" >
+			<label for="EYES1"> <?php echo xlt('Vision Changes');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="EYES2" name="EYES2" type="checkbox" value="on" >
+			<label for="EYES2"> <?php echo xlt('Floaters');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="EYES3" name="EYES3" type="checkbox" value="on" >
+			<label for="EYES3"> <?php echo xlt('Diplopia');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="EYES4" name="EYES4" type="checkbox" value="on" >
+			<label for="EYES4"> <?php echo xlt('Blurry Vision');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="EYES5" name="EYES5" type="checkbox" value="on" >
+			<label for="EYES5"> <?php echo xlt('Eye Pain');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="EYES6" name="EYES6" type="checkbox" value="on" >
+			<label for="EYES6"> <?php echo xlt('Photophobia');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="EYES7" name="EYES7" type="checkbox" value="on" >
+			<label for="EYES7"> <?php echo xlt('Red Eyes');?></label>
+		</span>
+		</TD>
+		</TR>
+		<TR>
+		<TD>
+		<?php echo xlt('Note abnormalities:');?> 
+					<TEXTAREA NAME="EYES_notes" ROWS=3 COLS=25 WRAP=SOFT STYLE="width: 2in; height: 0.21in"></TEXTAREA>
+					</TD>
+	</TR>	
+
+	
+	
+	<TR class="alt">
+	<TD>
+	<B><?php echo xlt('ENT:');?></B>
```

Comment:
I always got shot down for punctuation in the information to be translated it breaks the translation